cmd/serial-vault/main.go

```diff
@@ -27,10 +27,17 @@ import (
 	"github.com/CanonicalLtd/serial-vault/datastore"
 	"github.com/CanonicalLtd/serial-vault/service"
 	svlog "github.com/CanonicalLtd/serial-vault/service/log"
+	"github.com/CanonicalLtd/serial-vault/service/sentry"
 	logging "github.com/op/go-logging"
 )
 
-func init() {
```

Comment:
cant use init() here because it runs before main(), but we need config initialized at this point